pyparsing/core.py

```diff
@@ -122,6 +121,13 @@ def enable_all_warnings(cls):
 # hide abstract class
 del __config_flags
 
+DebugActions = namedtuple("DebugActions", ["TRY", "MATCH", "FAIL"])
```

Comment:
Using a named tuple makes debugging a tiny bit easier: assigning names to the tuple indexes